Puzzlepart/prosjektportalen365
# Allowing a pre-defined template as input to the wizard #448

### Your checklist for this pull request


### Description

Solves #448 by looking for a value in a property bag named "pptemplate". If set, the wizard only retrieves the template with the same filename as the value of "pptemplate".

### How to test

This is a technical change with limited user effect. Jan - you don't have to test this ;)

1. Create a new project site
2. Add a property bag to the site - ""pptemplate" with a value of a template file, e.g. "Standardmal.txt"
3. Observe that when the wizard runs, only the chosen template can be applied

### Relevant issues (if applicable)

#448 

💔Thank you!
